python/cc/BUILD.bazel

```diff
@@ -0,0 +1,44 @@
+# Package for C/C++ specific functionality of the Python rules.
+
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("//python/private:current_py_cc_headers.bzl", "current_py_cc_headers")
+load("//python/private:util.bzl", "BZLMOD_ENABLED")
```

Comment:
@rickeylev 👍 - We needed this.  We need to file an issue to refactor everywhere to use this.

Reply:
Yeah, good idea. Filed https://github.com/bazelbuild/rules_python/issues/1295